modules/services/protobuf/src/main/proto/sync_api.proto

```diff
@@ -223,3 +223,9 @@ message PodcastRatingAddRequest {
 message PodcastRatingShowRequest {
     string podcast_uuid = 1;
 }
+
+message SupportFeedbackRequest {
+     string subject = 1;
+      string inbox = 2;
+      string message = 3;
+}
```

Comment:
Can you change the message to the following:
```
message SupportFeedbackRequest {
    string message = 1;
    string email = 2;
    string subject = 3;
    string debug = 4;
    string inbox = 5;
}
```
This is from the following location https://github.com/Automattic/pocket-casts-sync-api/blob/develop/api/modules/protobuf/src/main/proto/api.proto#L105

Reply:
I've just updated the code with this! [232b852](https://github.com/Automattic/pocket-casts-android/pull/2581/commits/232b8524ef3a7ed77f02b8bebf9cb089e9a87269)